src/Components/PersonRaw/PersonRow.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect } from 'react';
+
+import { getPeople } from '../../api/api';
+
+export const PersonRow = () => {
```

Comment:
```suggestion
export const PersonRows = () => {
```
You are mapping many rows inside this component (not only one row), so it should be called PersonRows (in plural)